docs/ref/models/options.txt

```diff
@@ -29,6 +29,10 @@ Available ``Meta`` options
 
         app_label = 'myapp'
 
+    .. versionadded:: 1.9
```

Comment:
Let's add a section to the bottom of this file called something like "Read-only ``Meta`` attributes".

Then we can follow the same format of the other options with a heading, `.. attribute:: Options.label`, and a description.